wa/workloads/angrybirds_rio/__init__.py

```diff
@@ -0,0 +1,29 @@
+#    Copyright 2013-2015 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from wa import ReventWorkload
+
+
+class AngryBirdsRio(ReventWorkload):
+
+    name = 'angrybirds_rio'
+    description = """
+    Angry Birds Rio game.
+
+    The sequel to the very popular Android 2D game.
+    """
+    package = 'com.rovio.angrybirdsrio'
+    activity = 'com.rovio.ka3d.App'
```

Comment:
Are these still necessary? IIRC, the new APK workload should auto-discover package and activity from APK info.